### ISSUE TITLE
Soft-locked multiplayer campaign save (Wasn't able to reproduce with the same save after reloading)

### ISSUE THREAD

**Issue by Rokvach** [Regalis11/Barotrauma#4087]
Reloading the save fixed the issue, so I wasn't able to reproduce the issue with the same save

Plausible way to reproduce:
1. Get a fresh campaign with a similar level distribution where a natural formation is next to the starting outpost which doesn't lead to anywhere else
2. Travel to the natural formation and see if you can enter it

I traversed to a Natural Formation which didn't lead to anything and upon getting to the end of the level and pressing the enter natural formation button I wasn't able to press Start mission when I selected the Outpost I just came from, teleporting back to the start of the level and docking to the starting Outpost doesn't load in to the outpost level but just docks the sub.

![image.png](https://images.zenhubusercontent.com/5c924117534a9b0a86cdddce/ba3fb40a-febd-46b9-9ed7-59ca7d6a1c4e)

**Comment by Regalis11** [comment]
Save file: 
[test 178.zip](https://github.com/Regalis11/Barotrauma/files/5385190/test.178.zip)

**Comment by Regalis11** [comment]
Could not reproduce. Tested by starting a new mp campaign with the same map seed and following the steps provided.

**Comment by Rokvach** [comment]
Nothing comes to mind as to what else I did before I teleported to the end of the level. I might of been in Freecamera mode for a moment when testing. But I loaded the round, fed the pets and played around with them and then used teleportsub end and tried to enter the natural formation but couldn't, then used teleportsub start and tried to dock to the starting outpost but after docking nothing happened, then I tried couple more times by teleporting to end and start and returned to multiplayer lobby and reloaded the save.
